app/Http/Transformers/ReactionTransformer.php

```diff
@@ -18,6 +18,15 @@ public function transform(Reaction $reaction)
         return [
             'northstar_id' => $reaction->northstar_id,
             'post_id' => (string) $reaction->post_id,
+            'term' => [
+                'name' => 'heart',
+                'id' => 641,
+                'total' => 1,
+            ],
+            'current_user' => [
```

Comment:
ohoh sorry you are totally right, Shae! Sorry I thought this was in the Reportback Transformer - I didn't look closely enough and I think this is from old code when I was fidgeting with other stuff!